Makefile

```diff
@@ -26,6 +26,14 @@ core_static:
 	go generate internal/notification/statik/generate.go
 	go generate internal/statik/generate.go
 
+.PHONY: core_generate_all
```

Comment:
I had the same doubt. I settled for this because the `statik` generation is part of the CI build and all the other generating stuff, like mock, enumer and stringer should be ran locally by devs and the resulting files committed to git.

For example, if this steps are merged:

A dev might change an interface. gomock is not typesafe, it uses variadic arguments to `EXPECT` functions. The dev does not run `make core_generate_all` locally, however the pipeline will as part of the build. Unit tests in the pipeline would pas, but the updated generated files are never committed.

Dev2 pulls zitadel and runs unit tests locally and see that they fail on something he never touched. The gomock files are out-of-date.

Reply:
I understand, I think we should have a look at git hooks in the future for that problem but for now it's ok then